src/backends/pyplot.jl

```diff
@@ -1183,6 +1183,12 @@ function py_add_legend(plt::Plot, sp::Subplot, ax)
                         color = py_color(_cycle(series[:fillcolor],1)),
                         linewidth = py_dpi_scale(plt, 4)
                     )
+                elseif series[:seriestype] == :path
+                    PyPlot.plt[:Line2D]((0,1),(0,0),
+                        color = py_color(_cycle(series[:fillcolor],1)),
+                        linewidth = py_dpi_scale(plt, 1),
+                        marker = py_marker(series[:markershape])
```

Comment:
maybe adding
```julia
edgecolor = py_markerstrokecolor(series)
facecolor = py_markercolor(series)
```
could fix the marker colors here. I'm not sure, though.

Reply:
Hmm, I tried these and I get the error:
```
Error showing value of type Plots.Plot{Plots.PyPlotBackend}:
ERROR: PyError (ccall(@pysym(:PyObject_Call), PyPtr, (PyPtr, PyPtr, PyPtr), o, arg, kw)) <type 'exceptions.AttributeError'>
AttributeError(u'Unknown property edgecolor',)
  File "/Users/jackdevine/.julia/v0.6/Conda/deps/usr/lib/python2.7/site-packages/matplotlib/lines.py", line 431, in __init__
    self.update(kwargs)
  File "/Users/jackdevine/.julia/v0.6/Conda/deps/usr/lib/python2.7/site-packages/matplotlib/artist.py", line 885, in update
    for k, v in props.items()]
  File "/Users/jackdevine/.julia/v0.6/Conda/deps/usr/lib/python2.7/site-packages/matplotlib/artist.py", line 878, in _update_property
    raise AttributeError('Unknown property %s' % k)
```
"Unknown property edgecolor"

I think that the problem is that I am using `PyPlot.plt[:Line2D]`, maybe I should try using something more specific.

Reply:
markeredgecolor and markerfacecolor could work with Line2D, cf. https://matplotlib.org/devdocs/api/_as_gen/matplotlib.lines.Line2D.html

Reply:
Thanks! That works :)

Reply:
Great!